packages/workspace/src/command-line/format.ts

```diff
@@ -26,8 +26,10 @@ const PRETTIER_EXTENSIONS = [
 export function format(command: 'check' | 'write', args: yargs.Arguments) {
   let patterns: string[];
 
+  const { nxArgs } = splitArgsIntoNxArgsAndOverrides(args);
+
   try {
-    patterns = getPatterns(args as any);
+    patterns = getPatterns(nxArgs as any);
```

Comment:
@FrozenPandaz I'm not sure how this fixes the issue. `args` is a super set of `nxArgs`.

Reply:
`args` doesn't have the default values processing that `splitArgsIntoNxArgsAndOverrides` provides.

`yarn format` used to be equal to `yarn format --base master --head HEAD` but without that parsing, it was not the case. This makes it the case again.